engine/render/src/render/constant.cpp

```diff
@@ -157,17 +162,109 @@ void RemoveNamedConstant(HNamedConstantBuffer buffer, dmhash_t name_hash)
     uint32_t remaining = buffer->m_Values.Size() - (values_index + num_values);
 
     dmVMath::Vector4* p_next = p_current + num_values;
-    memmove(p_current, p_next, remaining); // if it's the last item, then "remaining" will be 0
+    memmove(p_current, p_next, remaining * sizeof(dmVMath::Vector4)); // if it's the last item, then "remaining" will be 0
```

Comment:
I think this was a bug previously, memmove expects bytes and remaining is just number of items to move